src/core/server/http/http_server.mocks.ts

```diff
@@ -182,9 +36,9 @@ const createToolkitMock = (): ToolkitMock => {
 };
 
 export const httpServerMock = {
-  createKibanaRequest: createKibanaRequestMock,
-  createRawRequest: createRawRequestMock,
-  createResponseFactory: createResponseFactoryMock,
+  createKibanaRequest: mockRouter.createKibanaRequest,
+  createRawRequest: hapiMocks.createRequest,
```

Comment:
I was wondering where the `hapiMocks.createRequest` came from and now it makes total sense.

Reply:
Yea, sorry I kinda added that after your review, but yea in term of separation it makes sense ihmo, it helps managing the mock dependency we're having in this domain